LambdaEngine/Source/Engine/EngineConfig.cpp

```diff
@@ -31,9 +37,7 @@ namespace LambdaEngine
 
 		s_ConfigDocument.ParseStream(inputStream);
 
-		fclose(pFile);
-
-		return true;
+		return fclose(pFile) == 0;
 	}
 
 	bool EngineConfig::WriteToFile()
```

Comment:
Would be safer if `EngineConfig::WriteToFile()` got updated as ReadToFile. File path for writing is still hardcoded.

Reply:
Good spot, fixed.